src/components/Home.js

```diff
@@ -10,8 +10,10 @@ const Home = () => {
     <div>
       <section className='px2 bg-blue-white'>
         <div className='py7 container relative'>
-          <h1 className='mt0 pb1 border-bottom'>National, state, and local crime data</h1>
-          <p className='m0 col-10 h3 serif'>
+          <h1 className='mt0 pb1 border-bottom border-red-bright'>
+            National, state, and local crime data
+          </h1>
+          <p className='m0 col-10 fs-b1 serif'>
```

Comment:
The single letter abbreviations already introduce a learning curve, though if they all are shortened versions of CSS selectors or properties I think that is manageable. If we start introducing our own "semantic" abbreviations I think it gets too confusing.

What do you think about `fs1` or `body-fs1` or `fs1-body`?